tests/test_db_runtime_security.py

```
@@ -752,6 +807,87 @@ def test_close_handles_connection_errors(self):
+            if not allow_release.wait(timeout=1.0):
+                release_error.append("test never allowed release_activity to continue")
+                return
+            with manager._connections_lock:
+                manager._active_sync_count = 0
+                manager._active_sync_condition.notify_all()
+
+        def close_manager() -> None:
+            close_started.set()
+            try:
+                manager.close()
+            except Exception as err:  # pragma: no cover - defensive
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)